src/euklid.py

```diff
@@ -0,0 +1,13 @@
+"""
+Даны два натуральных числа. Вычислите их наибольший общий делитель
+при помощи алгоритма Евклида (мы не знаем функции и рекурсию).
+"""
+
+num1 = int(input("Enter the first number:"))
+num2 = int(input("Enter the second number:"))
+while num1 != 0 and num2 != 0:
+    if num1 > num2:
+        num1 = num1 % num2
+    else:
+        num2 = num2 % num1
+print(num1 + num2)
```

Comment:
it's a cheat. algorithm doesn't expect result in sum. 
you can use ternary operator instead or rewrite algorithm